x-pack/plugins/synthetics/public/apps/uptime_app.tsx

```diff
@@ -124,7 +124,7 @@ const Application = (props: UptimeAppProps) => {
                     <UptimeSettingsContextProvider {...props}>
                       <UptimeThemeContextProvider darkMode={darkMode}>
                         <UptimeStartupPluginsContextProvider {...startPlugins}>
-                          <UptimeIndexPatternContextProvider data={startPlugins.data}>
```

Comment:
Can we rename the file `x-pack/plugins/uptime/public/contexts/uptime_index_pattern_context.tsx` to match the `data_view` name too?